taxi/views.py

```diff
@@ -28,6 +30,20 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def assign_driver_to_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    car.drivers.add(request.user)
```

Comment:
The `assign_driver_to_car` function directly adds the `request.user` to the car's drivers. Ensure that the `request.user` is indeed a `Driver` instance to prevent potential errors.